modules/tinymce/src/core/main/ts/api/commands/TableCommands.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
I added this file when rebasing and registered it in [Commands.ts](https://github.com/tinymce/tinymce/pull/7583/files#diff-9e36a5f6df6c7111295466824a34b6a2686bffa1983e57c43aa28bacd9b40361R61)